elyra/pipeline/handlers.py

```diff
@@ -178,6 +189,27 @@ async def get(self, runtime_type):
         await self.finish(palette_json)
 
 
+class PipelinePropertiesHandler(HttpErrorMixin, APIHandler):
+    """Handler to expose method calls to retrieve pipeline properties"""
+
+    @web.authenticated
+    async def get(self, runtime_type):
+        self.log.debug(f"Retrieving pipeline components for runtime type: {runtime_type}")
+
+        runtime_processor_type = get_runtime_processor_type(runtime_type, self.log, self.request.path)
+        if not runtime_processor_type:
+            raise web.HTTPError(400, f"Invalid runtime type '{runtime_type}'")
+
+        # Get pipeline properties json
+        jinja_loader = PackageLoader("elyra", "templates/pipeline")
+        template = Environment(loader=jinja_loader).get_template("pipeline_properties_template.jinja2")
```

Comment:
@akchinSTC @kiersten-stokes just in case you missed this black warning

Reply:
im not seeing a black warning for this line, weird

Reply:
I found the same thing.  The warning is actually the code-scan (that is viewable from the `Files changed` tab) stating that `autoescape` is false (the default) and should be specified in the `Environment()` constructor as `True`.
```suggestion
        template = Environment(loader=jinja_loader, autoescape=True).get_template("pipeline_properties_template.jinja2")
```

Reply:
if i remember correctly, using `autoescape=True` results in the template being rendered with encoded characters to replace punctuation. We would need to wrap all the content with {{ content | safe }} to avoid read issues coming from the front end.